tools/sdk/uxdProtocol/uxdClient.ts

```diff
@@ -12,14 +12,15 @@ const DEFAULT_WALLET_PROVIDER = 'https://sollet.io'
 const CLUSTER = 'devnet'
 
 export const uxdClient = (
```

Comment:
UXD client has nothing to do with Controller. Please separate and find a generic way to instantiate logical entities like Controller/Depository. To initialize controller in unrelated instruction, use an input for the on chain adress and please fetch the account info to fill what's required.